smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolUnitTestResponseGenerator.kt

```diff
@@ -139,44 +139,96 @@ open class HttpProtocolUnitTestResponseGenerator protected constructor(builder:
     }
 
     protected open fun renderAssertions(test: HttpResponseTestCase, outputShape: Shape) {
-        val members = outputShape.members().filterNot { it.hasTrait(HttpQueryTrait::class.java) }
-        val path = ".properties".takeIf { outputShape.hasTrait<ErrorTrait>() } ?: ""
-        renderMemberAssertions(writer, test, members, model, symbolProvider, "expected$path", "actual$path")
+        findShapesRequiringEquatable(outputShape)
+        writer.write("XCTAssertEqual(actual, expected)")
```

Comment:
High level:
- Find the model types that need to have Equatable added to them so that the output is equatable, and add them.
- Insert a simple comparison of the output type since it will now be Equatable.